### PR TITLE
[16.0][MIG] account_invoice_transmit_method

### PR DESCRIPTION
**Pull request by alexis-via**

### CODE REVIEW

**Comment by alexis-via** [comment]
Ready for review ; tests all green. I added a sequence field on transmit method, so that the user can control the order.

**Review by jguenat — APPROVED** [review]
Code & functional review
LGTM

File: account_invoice_transmit_method/models/account_move.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            ]
+        """,
+    )
+    transmit_method_code = fields.Char(
+        string="Transmission Method Code",
+        related="transmit_method_id.code",
+        help="Technical field used for UX purposes",
+    )
+
+    @api.depends("partner_id", "move_type")
```
Comment (flotho) [inline account_invoice_transmit_method/models/account_move.py:33]: Hi for my concern, is this the replacement of an onchange method ?

**Review by flotho — APPROVED** [review]
code review LGTM,

Thanks @alexis-via

File: account_invoice_transmit_method/models/account_move.py (hunk 1 above)
Comment (alexis-via) [reply account_invoice_transmit_method/models/account_move.py:33]: yes, it is

**Comment by BT-apadron** [comment]
Can you please tell me when this PR is going to merge?

**Pull request merged**